From abe19fcddf23c61c4ba5f4bc627cf70ea74ac21f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ralf=20Cors=C3=A9pius?= <corsepiu@fedoraproject.org>
Date: Sat, 19 Jan 2013 08:28:35 +0100
Subject: [PATCH 1/6] Add Fedora configuration.

---
 etc/RT_SiteConfig.pm | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/etc/RT_SiteConfig.pm b/etc/RT_SiteConfig.pm
index 4d2ef35..0c09743 100755
--- a/etc/RT_SiteConfig.pm
+++ b/etc/RT_SiteConfig.pm
@@ -10,13 +10,14 @@
 #
 # The converse is also true, if this file isn't valid perl, you're
 # going to run into trouble. To check your SiteConfig file, use
-# this comamnd:
+# this command:
 #
-#   perl -c /path/to/your/etc/RT_SiteConfig.pm
+#   perl -c /etc/rt/RT_SiteConfig.pm
 #
 # You must restart your webserver after making changes to this file.
 
-Set( $rtname, 'example.com');
+Set($rtname, 'example.com');
+Set($WebPath, '/rt');
 
 # You must install Plugins on your own, this is only an example
 # of the correct syntax to use when activating them.
-- 
1.9.3

