From d5862375f6521cc7e0ba10bcd69621b375d95b1f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ralf=20Cors=C3=A9pius?= <corsepiu@fedoraproject.org>
Date: Tue, 18 Mar 2014 14:47:19 +0100
Subject: [PATCH 3/6] Broken test-dependencies.

---
 sbin/rt-test-dependencies.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sbin/rt-test-dependencies.in b/sbin/rt-test-dependencies.in
index 575c916..9a8731a 100644
--- a/sbin/rt-test-dependencies.in
+++ b/sbin/rt-test-dependencies.in
@@ -197,7 +197,7 @@ CGI 3.38
 CGI::Cookie 1.20
 CGI::Emulate::PSGI
 CGI::PSGI 0.12
-Class::Accessor::Fast
+Class::Accessor::Fast 0.34
 Crypt::Eksblowfish
 CSS::Minifier::XS
 CSS::Squish 0.06
-- 
2.5.0

